src/app/components/tag/TagInput.js

```diff
@@ -95,7 +108,7 @@ class TagInput extends React.Component {
 
   render() {
     return (
-      <div style={{ padding: units(2) }}>
+      <div style={paddingStyle}>
```

Comment:
The objective is to replace the inline styling usage.
E.g. Instead of `<div style={{ padding: units(2) }}>`, we would have `<Box p={2}>`.